orchagent/aclorch.cpp

```diff
@@ -2243,6 +2243,7 @@ void AclOrch::init(vector<TableConnector>& connectors, PortsOrch *portOrch, Mirr
             platform == CISCO_8000_PLATFORM_SUBSTRING ||
             platform == MLNX_PLATFORM_SUBSTRING ||
             platform == BFN_PLATFORM_SUBSTRING  ||
+            platform == CLX_PLATFORM_SUBSTRING  ||
```

Comment:
Failing build on this defintion - CLX_PLATFORM_SUBSTRING. Please make sure associated PR is merged

Reply:
@prsunny Hi Prince, looks like the pipeline failure was fixed, could you please kindly review this PR? Thanks in advance.